package.json

```diff
@@ -41,7 +36,82 @@
   "bugs": {
     "url": "https://github.com/absolute-version/commit-and-tag-version/issues"
   },
+  "eslintConfig": {
```

Comment:
I'm curious what your reasoning for moving this into the package json is - happy to accept it if it's an improvement, but in general for open source I prefer separate config files because:

1) You can tell from the file structure what is configured and what isn't (and, what tools are used)
2) The package json is easier to read, because it's mostly deps and scripts

Reply:
It's the pattern we used in the last place I was going lots of Node.

Happy to separate out into a Jest config and eslint file, does make sense to be separated from your reasoning above